content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (India region)"
+date: 2022-01-10
+slug: meet-our-contributors-india-ep-01
```

Comment:
```suggestion
layout: blog
title: "Meet Our Contributors - APAC (India region)"
date: 2022-01-10T12:00:00+0000
slug: meet-our-contributors-india-ep-01
canonicalUrl: https://kubernetes.dev/blog/2022/01/10/meet-our-contributors-india-ep-01/
```

(delay until the other post is published, and mark that one as the canonical URL)

Reply:
@sftim, looks like the [other post](https://kubernetes.dev/blog/2022/01/10/meet-our-contributors-india-ep-01/) is live on the site now. I've updated the PR with the changes pointed above.

Reply:
Oops, that's a bug!

Reply:
Yea, I was wondering the same. 😅 The date for publish is Jan 10.